Fix redefinition of htonll and ntohll (with unknown consequences) on
OS X 10.10 and later.
https://salsa.debian.org/clint/fakeroot/-/merge_requests/17
--- message.h.orig	2021-11-07 19:33:39.000000000 -0600
+++ message.h	2022-05-23 02:50:41.000000000 -0500
@@ -26,8 +26,8 @@
 # endif
 #endif
 
-/* On Solaris, use the native htonll(n)/ntohll(n) */
-#if !defined(sun) && !defined(_NETINET_IN_H)
+/* If available, use the native htonll(n)/ntohll(n */
+#if !defined(htonll) || !defined(ntohll)
 #if __BYTE_ORDER == __BIG_ENDIAN
 # define htonll(n)  (n)
 # define ntohll(n)  (n)
